CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2018-01-05 - Supported Release 1.4.1
+
+The DSC module received a few bug fixes related to the PowerShell handler and a security fix in the form of enabling users to specify credentials as sensitive.
```

Comment:
Since this is a data type, I think we want to mark it uppercase.

Also, could be good to link it to https://puppet.com/docs/puppet/latest/lang_data_sensitive.html